charts/harmony-chart/Chart.yaml

```diff
@@ -52,3 +52,9 @@ dependencies:
   version: "v0.29.2"
```

Comment:
you need to bump the version of the harmony chart to trigger a new release

Reply:
Thank you! Would you like me to bump before or after review?